packages/Stylers/src/helpers.scss

```diff
@@ -6,9 +6,39 @@
 
 // Utilities
 
+@function div($dividend, $divisor, $precision: 12) {
```

Comment:
Did you reference an algorithm you found online for this? If so, can we include a comment with the link?